docs/source/parallelism.mdx

```diff
@@ -39,11 +39,11 @@ The following is the brief description of the main concepts that will be describ
 5. Sharded DDP - is another name for the foundational ZeRO concept as used by various other implementations of ZeRO.
 
 
-## Data Parallel
+## Data Parallelism
```

Comment:
I think `parallelism` is better than `parallel`. how about you? - @stas00

Reply:
In addition, Tensor parallelism section is written as `parallelism`, not `parallel`.